packages/veritone-widgets/src/redux/modules/mediaDetails/saga.js

```diff
@@ -679,23 +685,38 @@ function* createFileAssetSaga(
   sourceData,
   fileData
 ) {
+  if (sourceData.sourceEngineId) {
```

Comment:
Move this to watchSaveAssetData before yielding to createFileAssetSaga.
createFileAssetSaga should be more generic. We use it to create plain/text asset for bulk transcript edit.